_quarto.yml

```diff
@@ -237,10 +250,6 @@ website:
           contents:
             - docs/overview.qmd
             - docs/user-interfaces.qmd
-        - section: "<span class='emoji-icon'>💻</span>  __Workflow__"
-          contents:
-            - docs/install-create-run.qmd
-            - docs/debug.qmd
```

Comment:
Could remove these files now that they've been effectively moved to new filles?

Reply:
<img width="1054" alt="image" src="https://github.com/user-attachments/assets/4120ed1e-f3b7-40d0-83a0-9b625cd1ece0" />

I might be missing some, but they show as deleted in the diff

Reply:
Ah sorry, somehow missed that